tests/core/chain-object/test_gas_estimation.py

```diff
@@ -23,18 +30,18 @@ def chain(chain_without_block_validation):
     'should_sign_tx', (True, False),
 )
 @pytest.mark.parametrize(
-    'data, gas_estimator, to, on_pending, expected',
+    'data, gas_estimator, to, on_pending, pre_instanbul_expected, post_constantinople_expected',
```

Comment:
This part is tripping me up, but I didn't quickly think of a better option. This approach going to get ugly quickly, after one or two more changes to this gas schedule.

Maybe we can try pairing the expected result with the VM, like:
```
( ... (ConstantinopleVM, 21680), (PetersburgVM, 21680), (IstanbulVM, 21160), ...)
```
(Also note that the last one would be post petersburg, instead of constantinople)